opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -23,57 +23,59 @@
     MetricsExporter,
 )
 from opentelemetry.sdk.metrics.export.aggregate import Aggregator
-from opentelemetry.sdk.metrics.export.batcher import UngroupedBatcher
+from opentelemetry.sdk.metrics.export.batcher import Batcher
 from opentelemetry.sdk.metrics.export.controller import PushController
+from opentelemetry.sdk.metrics.view import (
+    ViewData,
+    ViewManager,
+    get_default_aggregator,
+)
 from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.util import get_dict_as_key
 from opentelemetry.sdk.util.instrumentation import InstrumentationInfo
 
 logger = logging.getLogger(__name__)
 
 
-def get_labels_as_key(labels: Dict[str, str]) -> Tuple[Tuple[str, str]]:
-    """Gets a list of labels that can be used as a key in a dictionary."""
-    return tuple(sorted(labels.items()))
-
-
 class BaseBoundInstrument:
     """Class containing common behavior for all bound metric instruments.
 
     Bound metric instruments are responsible for operating on data for metric
     instruments for a specific set of labels.
 
     Args:
-        value_type: The type of values for this bound instrument (int, float).
-        enabled: True if the originating instrument is enabled.
-        aggregator: The aggregator for this bound metric instrument. Will
-            handle aggregation upon updates and checkpointing of values for
-            exporting.
+        labels: A set of labels as keys that bind this metric instrument.
+        metric: The metric that created this bound instrument.
     """
 
     def __init__(
-        self,
-        value_type: Type[metrics_api.ValueT],
-        enabled: bool,
-        aggregator: Aggregator,
+        self, labels: Tuple[Tuple[str, str]], metric: metrics_api.MetricT,
     ):
-        self.value_type = value_type
-        self.enabled = enabled
-        self.aggregator = aggregator
+        self._labels = labels
+        self._metric = metric
```

Comment:
Could we avoid using the term "metric" to describe an instrument? The API specs very carefully avoid using metric as a noun, the only place we use that is to describe the result in a protocol such as OTLP. "Metric" is being used to describe the output, in other words, and "Instrument" is the thing we use for input.

Reply:
A naming pass will be done outside of this PR to update all the names to the specs.